packages/smarthr-ui/src/components/ResponseMessage/ResponseMessage.tsx

```diff
@@ -9,8 +9,12 @@ import {
   type ComponentProps as IconProps,
   WarningIcon,
 } from '../Icon'
+import { Text } from '../Text'
 
-type Props = PropsWithChildren<VariantProps<typeof classNameGenerator>> & Omit<IconProps, 'text'>
+type Props = PropsWithChildren<VariantProps<typeof classNameGenerator>> &
+  Omit<IconProps, 'text' | 'size'> & {
+    size?: VariantProps<typeof Text>['size']
```

Comment:
VariantPropsはtailwind-variantsのtvからPropsを生成する型なので、reactのコンポーネントのPropsを取り出す用途である `ComponentPropsWithoutRef` を使いたいです！

```suggestion
    size?: ComponentPropsWithoutRef<typeof Text>['size']
```

Reply:
`ComponentPropsWithoutRef` を使いつつ、u さん指摘の通り size を絞ろうとして

`size?: Pick<ComponentPropsWithoutRef<typeof Text>['size'], 'XS' | 'S' | 'M'>`

としてみたんですが、

<img width="706" height="111" alt="スクリーンショット 2025-10-02 12 03 14" src="https://github.com/user-attachments/assets/f99b5c74-8a75-4c00-84b9-eead4c4ec3d5" />

<img width="1201" height="365" alt="スクリーンショット 2025-10-02 12 02 43" src="https://github.com/user-attachments/assets/c1b3bf6f-1e95-41a4-8106-a15572dcf065" />
型エラーになってしまう……

Pick使わずに XS S M に絞れたりするのかしら